res/mkn/mpi_asan.yaml

```
@@ -0,0 +1,19 @@
+  mod-repo: git@github.com:mkn-mod/
+env: |
+  ASAN_OPTIONS=detect_leaks=0
+compiler:
+  mask:
+    g++:
+      mpicxx
+file:
+  - type: cpp:cxx:cc
+    archiver: ar -cr
+    compiler: ccache mpicxx ${cxx_flags}
+    linker: mpicxx ${cxx_flags} 
```

yamllint

[error] trailing spaces

(trailing-spaces)